vispy/app/application.py

```diff
@@ -106,7 +105,9 @@ def is_interactive(self):
 
         # IPython does not set sys.flags when -i is specified, so first
         # check it if it is already imported.
-        if '__IPYTHON__' not in dir(six.moves.builtins):
+        try:
+            get_ipython()
+        except NameError:
```

Comment:
Are you sure this behaves the same? Wouldn't this work:

```
import builtins
if hasattr(builtins, '__IPYTHON__'):
```

Maybe these are essentially the same?

Reply:
AFAIK, it should however you are correct, we can keep the original logic. I somehow messed up something in my installation when doing the checks to update the code.